weaver/tests/functional/application-packages/WPS1/DeployProcess_CatalogSearch.json

```diff
@@ -38,8 +44,8 @@
     },
     "executionUnit": [
         {
-            "href": "catalog_search.cwl"
+            "href": "weaver/tests/functional/application-packages/WPS1/catalog_search.cwl"
         }
     ],
-    "deploymentProfileName": "http://www.opengis.net/profiles/eoc/workflow"
+    "deploymentProfileName": "http://www.opengis.net/profiles/eoc/wpsApplication"
```

Comment:
I know that the only options that we have right are "workflow" and "dockerizedApplication", so wpsApplication seems to make sense... But are we actually using that value?

Reply:
yes, see your answer ;)